docs/en/operations/system_tables.md

```diff
@@ -273,10 +275,68 @@ SELECT * FROM system.metrics LIMIT 10
 │ DistributedSend            │     0 │ Number of connections to remote servers sending data that was INSERTed into Distributed tables. Both synchronous and asynchronous mode.                                                          │
 └────────────────────────────┴───────┴──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
 ```
+
 **See Also**
 
 - [system.asynchronous_metrics](#system_tables-asynchronous_metrics) — Contains periodically calculated metrics.
 - [system.events](#system_tables-events) — Contains a number of events that occurred.
+- [system.metric_log](#system_tables-metric_log) — Contains a history of metrics values from tables `system.metrics` и `system.events`.
+- [Monitoring](monitoring.md) — Base concepts of ClickHouse monitoring.
+
+## system.metric_log {#system_tables-metric_log}
+
+Contains history of metrics values from tables `system.metrics` and `system.events`, periodically flushed to disk.
+For turn on metrics history collection on `system.metric_log`, create `/etc/clickhouse-server/config.d/metric_log.xml` with following content:
```

Comment:
Probably
```suggestion
To turn on metrics history collection on `system.metric_log`, create `/etc/clickhouse-server/config.d/metric_log.xml` with following content:
```
or even
```suggestion
Metrics history collection is enabled in server settings.
For example create `/etc/clickhouse-server/config.d/metric_log.xml` with following content or add this block to other server configuration file:
```

Reply:
system.metric_log doesn't enable by default, so a first suggestion is chosen